ui/pages/confirmations/selectors/confirm.ts

```diff
@@ -28,15 +28,9 @@ export function pendingConfirmationsSortedSelector(
     .sort((a1, a2) => a1.time - a2.time);
 }
 
-const internalLatestPendingConfirmationSelector = createSelector(
+export const oldestPendingConfirmationSelector = createSelector(
```

Comment:
Do we definitely to keep the `createDeepEqualSelector` here?

The recent Redux state PR should only update `pendingApprovals` when it actually changes, but if it does, then we'll get a fresh object reference and re-render here unnecessarily?

These renders are critical since they can impact everything using `useConfirmContext`.